docs/topics/circuit-breaker-mission-basic-interaction-spring-boot-tomcat.adoc

```diff
@@ -0,0 +1,104 @@
+= Interact with the Unmodified {SpringBoot} Booster
+
+Once you have the {SpringBoot} booster deployed, you have the following services running:
+
+//{app-name} = springboot-cb
+
+`{app-name}-name`::
+Exposes the following endpoints:
+
+* the `/api/name` endpoint, which returns a name when this service is working, and an error when this service is set up to demonstrate failure.
+
+* the `/api/state` endpoint, which controls the behavior of the `/api/name` endpoint and determines whether the service works correctly or demonstrates failure.
+
+`{app-name}-greeting`::
+Exposes the following endpoints:
+
+* the `/api/greeting` endpoint that you can call to get a personalized greeting response.
++
+Calling the `/api/greeting` endpoint also calls the `/api/name` endpoint of the `{app-name}-name` service.
+This call happens behind the scenes and is protected by the Circuit Breaker.
```

Comment:
@Ladicek sorry, just saw this and already merged. @inoxx03 feel free to file a follow up PR and i'll merge it.